src/user_management/user.model.js

```diff
@@ -1,11 +1,39 @@
+import bcrypt from 'bcrypt'
+const saltRounds = 10;
+
 export default (sequelize, DataTypes) => {
   const User = sequelize.define('user', {
     id: {
       type: DataTypes.UUID,
       primaryKey: true,
       defaultValue: DataTypes.UUIDV4
+    },
+    firstName: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    lastName: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    email: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    password: {
+      type: DataTypes.STRING,
+      allowNull: false
     }
   })
 
+  User.prototype.validatePassword = async(password) => {
+    return await bcrypt.compare(password, this.password)
+  }
```

Comment:
- There's no need for an `await` if you're returning the promise directly.
- If you're not using `await` you have to remove the `async`
- Don't use arrow functions here!! The `this` keyword **will not be what you expect**. Arrow functions capture the `this` keyword from its enclosing scope. You need to use a regular `function () {}` so that the `this` keyword is the object on which the function is being called.